File: Java/untitled/src/main/java/p191_NumberOf1Bits.java
public class p191_NumberOf1Bits {
    public int hammingWeight(int n) {
        int sum = 0;
        while (n!=0){
            sum += n%2;
            n/=2;
        }
        return sum;
    }

    public static void main (String[] args){
        p191_NumberOf1Bits solver = new p191_NumberOf1Bits();
        System.out.println(solver.hammingWeight(11));
        System.out.println(solver.hammingWeight(128));
        System.out.println(solver.hammingWeight(2147483645));
    }
}

File: Csharp/Csharp/p191_NumberOf1Bits.cs
namespace Csharp;

public class p191_NumberOf1Bits
{
    public int hammingWeight(int n) {
        int sum = 0;
        while (n!=0){
            sum += n%2;
            n/=2;
        }
        return sum;
    }
    
    public static void Main()
    {
        p191_NumberOf1Bits solution = new p191_NumberOf1Bits();
        System.Console.WriteLine(solution.hammingWeight(11));
        System.Console.WriteLine(solution.hammingWeight(128));
        System.Console.WriteLine(solution.hammingWeight(2147483645));
    }
}